packages/stream-metadata/src/routes/spaceImage.ts

```diff
@@ -91,7 +106,10 @@ export async function fetchSpaceImage(request: FastifyRequest, reply: FastifyRep
 			},
 			'Failed to get encryption key or iv',
 		)
-		return reply.code(500).send('Failed to get encryption key or iv')
+		return reply
+			.code(500)
+			.header('Cache-Control', CACHE_CONTROL['5xx'])
```

Comment:
This should not be a 500, this is a validation error. I think it's a 422.

https://www.rfc-editor.org/rfc/rfc9110.html#name-422-unprocessable-content

Reply:
@brianathere done!